### ISSUE TITLE
Pixi.js v4 support

### ISSUE THREAD

**Issue by griest024** [kittykatattack/charm#10]
I figure we can keep a list of the stuff that breaks in v4 here.

- PIXI.ParticleContainer needs to be renamed to PIXI.particles.ParticleContainer

**Comment by LeXXik** [comment]
@kittykatattack Thank you for the library. Do you plan on adding support for the latest Pixi version?

I also found this fork, which adds support for v4:
https://github.com/dgzornoza/charm

**Comment by kittykatattack** [comment]
Yes, eventually.
But, as far as I know, it does work with the current version (if not, please list errors here)

**Comment by griest024** [comment]
@LeXXik check out [my fork](https://gitlab.com/griest/pexi/tree/master/src/charm).

**Comment by LeXXik** [comment]
@griest024 well, actually the original one works for me, as @kittykatattack mentioned. Perhaps I am not using the version breaking features? Not sure. But I will stick with this one. Thanks for the heads up though.

**Comment by griest024** [comment]
Yes that one change that I mentioned above was the only breaking change I could find, but my testing has not been extensive.

**Comment by birdkingz** [comment]
@griest024  I've tried to install via npm but I'm getting error `src\tween-path modules not found`. Please advise.
